sass/themes/sr/2018/components/single-msg/_single-msg.scss

```diff
@@ -0,0 +1,59 @@
+.single-msg.paragraph {
+
+  @include breakpoint($screen-md) {
+    .single-msg__outer-wrapper {
+      display: -webkit-flex;
+      display: flex;
```

Comment:
doesn't postcss add the vendor prefixes?

Reply:
One to investigate when I get back, not really a problem for now